config/samples/perf_v1alpha1_pgbench.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: perf.kubestone.xridge.io/v1alpha1
+kind: Pgbench
+metadata:
+  name: pgbench-sample
+spec:
+  image:
+    name: postgres:11.5-alpine
+  postgres:
+    host: postgres
+    port: 5432
+    user: admin
+    password: admin
```

Comment:
i'm not sure if it is possible, but would be nice to not to store plain-text passwords

Reply:
this is just an example, also: https://github.com/xridge/kubestone/issues/130